tutorials/gm-world.md

```diff
@@ -34,7 +34,7 @@ Rollkit uses the [Go programming language](https://go.dev/dl/). Here's how to in
 - **Linux or macOS**: Run the provided script:
 
   ```bash-vue
-  curl -sSL https://rollkit.dev/install-go.sh | sh -s {{constants.golangVersion}}
+  curl -sSL https://rollkit.dev/install-go.sh | bash -s {{constants.golangVersion}}
```

Comment:
can we use the same format as `bash -c "$(curl -sSL https://rollkit.dev/install-gm-rollup.sh)"` so that the script isn't saved locally, and also pass in the golang version/local da version?

Reply:
this can be done in a new PR